vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -161,6 +161,12 @@ bool TextViewManager::UpdateProperty(
   } else if (TryUpdateTextAlignment(textBlock, propertyName, propertyValue)) {
   } else if (TryUpdateTextTrimming(textBlock, propertyName, propertyValue)) {
   } else if (TryUpdateTextDecorationLine(textBlock, propertyName, propertyValue)) {
+    // Temporary workaround for bug in XAML which fails to flush old TextDecorationLine render
+    std::wstring text(textBlock.Text().c_str());
+    text.push_back('a');
```

Comment:
nit: setting to an empty string as in the original fix is perhaps slightly cleaner